observation_portal/proposals/notifications.py

```diff
@@ -14,6 +14,29 @@ def users_to_notify(requestgroup):
     ]
 
 
+def request_notifications(request):
```

Comment:
We should probably run this email template past Nikolaus for his approval.

Reply:
Should have mentioned in the review that I worked with him for the email subject and message